sdks/python/setup.py

```diff
@@ -93,6 +93,7 @@ def get_version():
     'protorpc>=0.9.1,<0.12',
     'python-gflags>=2.0,<4.0.0',
     'pyyaml>=3.10,<4.0.0',
+    'pyhamcrest>=1.9,<2.0',
```

Comment:
@robertwb is this the list of dependencies that gets installed when submitting a Dataflow job? Do we want pyhamcrest here?